sample/src/androidTest/java/com/schibsted/spain/barista/sample/WrappedViewClickTest.java

```diff
@@ -0,0 +1,46 @@
+package com.schibsted.spain.barista.sample;
+
+import android.support.test.rule.ActivityTestRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static com.schibsted.spain.barista.BaristaAssertions.assertDisplayed;
+import static com.schibsted.spain.barista.custom.PerformClickAction.clickUsingPerformClick;
+
+public class WrappedViewClickTest {
+
+  @Rule
+  public ActivityTestRule<WrappedViewActivity> activityTestRule =
+      new ActivityTestRule<>(WrappedViewActivity.class);
+
+  @Test
+  public void click_on_button() throws Exception {
```

Comment:
There are tons of tests written with the camel case style. If you prefer to start using other style, please, change all Barista's tests.